react/src/util/api-client.ts

```diff
@@ -20,15 +20,17 @@ export const getAddressDetails = async (
 
 export const getSatoshiBalance = async (
   address: string,
+  rootUrl = `https://api.paybutton.org/`,
```

Comment:
What do you think about moving that last `/` into the `fetch()` calls? Do you think that's better or worse?

Reply:
I think that's a much better way to do it. I'll get that changed asap.